go.mod

```diff
@@ -12,21 +12,25 @@ require (
 	github.com/prometheus/client_model v0.6.1
 	github.com/prometheus/common v0.60.1
 	github.com/prometheus/procfs v0.15.1
+	github.com/stretchr/testify v1.9.0
```

Comment:
We don't want to add extra dependency on purpose in client_golang for little value.